openmetadata-ui/src/main/resources/ui/src/constants/regex.constants.ts

```diff
@@ -52,3 +52,5 @@ export const HEX_COLOR_CODE_REGEX = /^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/;
 export const TASK_SANITIZE_VALUE_REGEX = /^"|"$/g;
 
 export const TIMESTAMP_UNIX_IN_MILLISECONDS_REGEX = /^\d{13}$/;
+
+export const ASTERISKS_MORE_THAN_TWICE_REGEX = /\*{3,}/;
```

Comment:
we should rethink matching this pattern, what if my password is `***mayur***` id never allows me to type full password


https://github.com/open-metadata/OpenMetadata/assets/39544459/842d797d-17c7-4e6d-90a2-d4eaf1f026bb